consensus/reactor.go

```diff
@@ -290,6 +290,11 @@ func (conR *ConsensusReactor) SetEventSwitch(evsw types.EventSwitch) {
 	conR.conS.SetEventSwitch(evsw)
 }
 
+// FastSync returns whether the consensus reactor is currently fast syncing
```

Comment:
actually the consensus reactor doesn't do the fast sync (the blockchain reactor does), so the comment should be updated